File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    score.sort((a,b)=>b-a);
    let sum = 0;
    for(let i = 0; i <score.length-m+1; i+=m) {
        sum += score[i+m-1];
    }
   
    return sum*m;
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):            
    sorted_score = sorted(score,reverse=True);
    result = 0;
    
    i = 0
    while i + m - 1 <= len(sorted_score) -1 :
        result += sorted_score[i + m - 1] 
        i += m

    return result * m